pkg/cmd/build/buildkit.go

```diff
@@ -125,15 +125,29 @@ func getSolveOpt(buildOptions *types.BuildOptions) (*client.SolveOpt, error) {
 	}
 
 	if buildOptions.Tag != "" {
-		opt.Exports = []client.ExportEntry{
-			{
-				Type: "image",
-				Attrs: map[string]string{
-					"name": buildOptions.Tag,
-					"push": "true",
+		// add additional tag if DevTag is defined
+		if buildOptions.DevTag != "" {
```

Comment:
I am not sure about this but `buildOptions.Tag` could have the same value as `buildOptions.DevTag`? and thus export twice with the same value?